JavaFXUserInterface/src/main/resources/css/style.css

```diff
@@ -67,4 +67,9 @@
 .input-error {
     -fx-border-color: red;
     -fx-border-width: 2px;
+}
+
+.splash-screen-bg {
+    -fx-background-color: rgb(22, 53, 96);
+    -fx-background-color: linear-gradient(to top right, rgba(22, 53, 96, 1) 0%, rgba(63, 143, 204, 1) 85%);;
```

Comment:
Is there a reason why there's two background-color attributes?

Reply:
The extra background-color attribute has been removed.